test/ussd_registration.test.js

```diff
@@ -38,7 +38,8 @@ describe("ussd_registration app", function() {
       send_sms_flow_id: "send-sms-flow-id",
       optout_flow_id: "optout-flow-id",
       popi_consent: "12-2021",
```

Comment:
Should we maybe remove this from the tests to make sure that it's not being used

Reply:
ah yes thanks

Reply:
Seems like we do need it for checking if the latest one has been accepted

Reply:
Ah, makes sense, we must just make a note somewhere then, to remember to update both when we need to change it.